charts/gardener/controlplane/charts/application/templates/clusterrole-seed-bootstrapper.yaml

```diff
@@ -11,11 +11,3 @@ rules:
   verbs:
   - create
   - get
-  - list
-  - watch
-- apiGroups:
-  - certificates.k8s.io
-  resources:
-  - certificatesigningrequests/seedclient
```

Comment:
Would someone be so kind to describe what that "seedclient" subresource in the CSR is for? Thanks!

At least I can see that the GCM creates a SubjectAccessReview for this subresource [here](https://github.com/gardener/gardener/blob/6fb1a16640e298eb22aed196d790ed842da7bbfb/pkg/controllermanager/controller/certificatesigningrequest/csr_autoapprove_control.go#L103)

Reply:
This is used to control whether GCM shall auto-approve CSRs created by gardenlets. Today, GCM has this privilege because it's cluster-admin in the whole garden (https://github.com/gardener/gardener/blob/2b8530028eee6425136887e7578bfac6c5f533a5/charts/gardener/controlplane/charts/application/templates/clusterrole-controller-manager.yaml#L12-L18).
In the future, when we further restrict these rules we might want to leave this decision up to the operators of Gardener - similar to how Kubernetes leaves it up to the operators whether KCM shall auto-approve CSRs by kubelets.